client/actions/index.js

```diff
@@ -11,6 +11,12 @@ export const SHOW_ERROR = 'SHOW_ERROR'
 export const HIDE_ERROR = 'HIDE_ERROR'
 export const SHOW_LOGIN = 'SHOW_LOGIN'
 export const SHOW_REGISTER = 'SHOW_REGISTER'
+export const SET_ENTRY_FILTER = 'SET_ENTRY_FILTER'
+
+export const EntryFilters = {
```

Comment:
Perhaps use https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Symbol

A
Also perhaps move to helper